packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -36,13 +36,24 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
       ...(p.compact && { padding: v.compactPadding }),
       ...(p.centered && { alignItems: 'center' }),
 
-      ...(p.actionable && {
-        cursor: 'pointer',
-        ':focus-visible': {
-          boxShadow: v.boxShadowFocus,
-        },
-        ':active': {
-          boxShadow: v.boxShadowPressed,
+      ...(p.actionable &&
+        !p.disabled && {
```

Comment:
Can we please not use negation in the styles condition. Just apply default styles and then override whatever needs to be overridden for disabled.